blivet/formats/fs.py

```diff
@@ -582,6 +582,13 @@ def loadModule(self):
         # also need to update the list of supported filesystems.
         update_kernel_filesystems()
 
+    @property
+    def systemMountpoint(self):
+        if not self.exists:
+            return None
+
+        return mountsCache.getMountpoint(self.device)
```

Comment:
I would like it slightly better if this had a comment/docstring, if the final line was:

```
return mountsCache.getMountpoint(self.device, getattr(self, "subvolspec", None))
```
and the overriding method in BTRFS class went away.